automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/azure/AzureRuntimeSpec.scala

```diff
@@ -95,6 +95,55 @@ class AzureRuntimeSpec
         )
         _ = monitorCreateResult.getStatus() shouldBe ClusterStatus.RUNNING
 
+        // Now stop the runtime
```

Comment:
I think we should do the stop/start in a separate test where we don't delete the runtime to shorten the length a bit

Reply:
One test creates and deletes a runtime, and this test creates, stops, starts, and deletes the runtime. I would prefer that we eliminate the first test and have this more complete runtime test instead. And, in the Azure Test readiness meeting this morning, they mentioned that we should be cleaning up resources (deleting the runtime). But that might have been talking about the E2E tests. 
So, I don't see the advantage of having two tests, where the second one is a complete duplicate of the first with a couple added steps.
This second test takes only a few minutes longer than the original one.

Reply:
If its not adding a significant amount of time, I'm fine with including it. As long as we make sure we don't approach the max of 1 hour, when the token for the tests will expire